web/app.js

```diff
@@ -1410,7 +1431,19 @@ const PDFViewerApplication = {
 
     const dispatchEventName = generateRandomStringForSandbox(objects);
     const calculationOrder = [];
-    scripting.createSandbox({ objects, dispatchEventName, calculationOrder });
+    scripting.createSandbox({
+      objects,
+      dispatchEventName,
+      calculationOrder,
+      docInfo: {
+        ...info,
+        metadata,
+        filesize: this.pdfDocumentProperties.maybeFileSize,
```

Comment:
This property is first of all not guaranteed to even be set at this point in time and/or it may not actually be correct. Secondly, it's not really correct to reach into that component in this way, since that property was never intended to be "public" in the first place.